DataFormats/HcalIsolatedTrack/src/classes_def.xml

```diff
@@ -36,4 +36,17 @@
   <class name="edm::Wrapper<reco::EcalIsolatedParticleCandidateCollection>"/>
   <class name="edm::reftobase::Holder<reco::Candidate, edm::Ref<std::vector<reco::EcalIsolatedParticleCandidate>,reco::EcalIsolatedParticleCandidate,edm::refhelper::FindUsingAdvance<std::vector<reco::EcalIsolatedParticleCandidate>,reco::EcalIsolatedParticleCandidate> > >" />
 
+  <class name="reco::HcalIsolatedTrackCandidate" ClassVersion="13">
+   <version ClassVersion="14" checksum="3313409542"/>
```

Comment:
``` ClassVersion="14" ``` should not be here.
Since this is the first time this class shows up in CMSSW, change the ``` ClassVersion="13"  ``` to ``` ClassVersion="2"  ```